frontend/lib/src/components/elements/ArrowVegaLiteChart/ArrowVegaLiteChart.tsx

```diff
@@ -48,10 +61,26 @@ const DEFAULT_DATA_NAME = "source"
  */
 const BOTTOM_PADDING = 20
 
+/**
+ * Debounce time for triggering a widget state update
+ * This prevents to rapid updates to the widget state.
+ */
+const DEBOUNCE_TIME_MS = 150
```

Comment:
I believe we use a similar pattern in the dataframe case, right? We should consider moving this to the WidgetState manager or some utils and re-use everywhere.

Reply:
Yeah, but I think there isn't that much to reuse. The debounce method is already a shared utility, and the value here is probably a bit more specific to the actual element interaction. Every interaction should probably choose a debounce value that feels good. E.g. the Slide uses a value of 200ms.

Reply:
if we unify this value, it can have unwanted side effects if it gets changed.

Reply:
yeah that makes sense to me